src/vmm_mad/remotes/lib/lxd/container.rb

```diff
@@ -384,6 +390,21 @@ def vnc(signal)
 
     private
 
+    # Deletes the switch port. Unlike libvirt, LXD doesn't handle this.
```

Comment:
Add return type to method description

Be consistent in returning variables. This method returns String, Boolean or nil